boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,947 @@
+#include <assert.h>
+#include <math.h>
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
```

Comment:
```suggestion
```

Reply:
Cannot be removed, keeping this is the right move. The compiler is just being too verboose/strict

Reply:
A workaround would be to make variables that we are doing bit manipulation on a uint32. I don't think it looks nice to have this.